portfolio/src/main/webapp/aboutMe.html

```diff
@@ -2,15 +2,27 @@
 <html>
   <head>
     <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
     <title>About Me</title>
   </head>
   <body>
-    <h1>About Me</h1>
-    <p>I am Isabelle Dittamo!</p>
-    <p>I am an intern at Google as a part of the STEP Internship for the Summer of 2020!</p>
-    <img src="images/googleGLogo.jpg" width="200"/>
-    <p>I'm also a rising 3rd year student at the Georgia Institute of Technology studying Computer Science with concentrations in Information Internetworks and Intelligence.</p>
-    <img src="images/gtLogoBuzz.png" width="200"/>
-    <p>At school, I am a Teaching Assistant in the College of Computing! I have previously TA'ed for the Introduction to Computing course.</p>
+    <div id="content">
+        <h1>About Me</h1>
+        <p>I am Isabelle Dittamo!</p>
+        <img src="images/profilePicture.jpg" height="300"/>
+        <p>I am an intern at Google as a part of the STEP Internship for the Summer of 2020!</p>
+        <img src="images/googleGLogo.jpg" width="200"/>
+        <p>I'm also a rising 3rd year student at the Georgia Institute of Technology studying Computer Science with concentrations in Information Internetworks and Intelligence.</p>
```

Comment:
Please limit lines to a character limit of ~100 characters/line. This improves readability during review and later for subsequent code readers. Here and elsewhere.

I couldn't find a way to put a character indicator in the editor for Cloud Shell (standard option in an IDE).

When you do this, follow this form:

```
<p>
   Text ...
   More text ...
</p>
```

Reply:
Got it! Fixed.